container/falcon-container-aks-implementation-guide.md

```diff
@@ -0,0 +1,282 @@
+# Implementation Guide for CrowdStrike Falcon-Container Sensor on Azure AKS Kubernetes cluster
+
+This guide works through creation of a new Kubernetes cluster, deployment of the Falcon-Container Sensor, and demonstration of detection capabilities of Falcon Container Workload Protection.
+
+No prior Kubernetes or Falcon knowledge is needed to follow this guide. First sections of this guide focus on creation of ACR Container registry and an AKS cluster, however, these sections may be skipped if you have access to an existing cluster.
+
+Time needed to follow this guide: 45 minutes.
+
+## Pre-requisites
+
+- Refer to the lastest documentation on the [falcon-container](https://falcon.crowdstrike.com/documentation/146/falcon-container-sensor-for-linux)
```

Comment:
For what purpose? There's a lot of text there. Anything in particular they should review ahead of time?